models/cesm.rst

```diff
@@ -1,4 +1,32 @@
 CESM model 
 ==========
 
-- CESM
+Brief description of model
+--------------------------
+The Community Earth System Model (CESM) is a state-of-the-art, fully coupled, global climate model that developed by the Climate and Global Dynamics Laboratory (CGD) at the National Center for Atmospheric Research (NCAR).
+
+The latest CESM version
+--------------------------
+The current CESM release is CESM 2.1.0 which is also used for CMIP6 experiments. The recent developments in CESM2 are explained here: http://www.cesm.ucar.edu/models/cesm2/whatsnew.html 
+
+How to download?
+--------------------------
+To download the current release :
+
+``git clone -b release-cesm2.1.0 https://github.com/ESCOMP/cesm.git my_cesm``
```

Comment:
We should probably mention https://github.com/NordicESMHub/cesm.git instead where we take our cime configurations.

Reply:
Thanks for spotting! It is fixed now.